src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -272,6 +272,7 @@ public function storeOutput(bool $isFailedJob = false): ?LoadTableQueue
                 $tableSystemMetadata,
                 $this->getStagingStorageOutput(),
                 $isFailedJob,
+                $this->storageConfig['output']['data_type_support'] ?? $this->component->getDataTypesSupport(),
```

Comment:
Je to nepříjemný, ale na tohle určitě bude potřeba test - tzn test na na to, že když zadáš output.data_type_support, tak to přebije nastavení komponenty